src/Spectre.libGenetic/CrossoverOperator.cpp

```diff
@@ -41,6 +42,16 @@ Individual CrossoverOperator::operator()(const Individual &first, const Individu
     {
         throw InconsistentChromosomeLengthException(first.size(), second.size());
     }
+    Individual child(first);
```

Comment:
Unnecessary copy. If you want to go this way, start with `Individual child = cross(first, second)` and then `while(!feasible) cross_again()`.

Reply:
done